samples/gatsby-cloud/src/index.css

```
@@ -8,22 +8,22 @@ body {
   vertical-align: baseline;
 }
 
-.preview-button {
+.site-button {
   height: 2.5rem;
   padding: 0 !important;
   font-size: 0.875rem !important;
   font-weight: 400 !important;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol !important;
 }
-.preview-button:focus {
+.site-button:focus {
   outline: 0;
   border: 1px solid #f1defa;
 }
 .powered-by {
   margin-bottom: 5%;
 }
 .powered-by p {
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol !important;
+  font-family: "Avenir Next W01", Helvetica, sans-serif !important;
```

We are actually moving away from Avenir, so I would ask that you put this one back.
```suggestion
  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol !important;
```